code-challenges.test.js

```diff
@@ -11,24 +11,57 @@
 // Run the file with the following command: $ yarn jest
 
 
-// --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+// 1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+
+//  describe("fib"), () => {
+//      it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence."), () => {
+//          expect(fib("fibLength1").toEqual([1, 1, 2, 3, 5, 8]))
+//      }
+//  }
+
+
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// -->Psuedocode
+// Create two sets of variables called fib and fib1.
+// Both variables will use a for loop that takes in the length of the index and out puts numbers from the Fibonacci sequence based on the length of the array provided on the const, either 6 or 10.
 
-// b) Create the function that makes the test pass.
+
+  var fib = [1, 1];
+  for(var i=fib.length; i<6; i++) {
+      fib[i] = fib[i-2] + fib[i-1];
+  }
+ var fib1 = [1, 1];
+ for(var i=fib1.length; i<10; i++) {
+     fib1[i] = fib1[i-2] + fib1[i-1];
+ }
+ console.log(fib, fib1); 
```

Comment:
The logic is there, but let's try to write this into a function for future assessments.  Your test won't run if there is no function declared as that is where the function is invoked.  I like to think of a function as a wrapper that you can wrap your code block in.  I highly recommend taking a look at this power point on [functions](https://docs.google.com/presentation/d/1qsvBc8dYt0gHb6jBZZEXeVWWY9bw-N-5eLkU3uxegvk/edit#slide=id.p) that helps break it down really clearly.